components/usage/pkg/db/usage_test.go

```diff
@@ -322,3 +322,37 @@ func TestListBalance(t *testing.T) {
 		CreditCents:   -50,
 	})
 }
+
+func TestGetBalance(t *testing.T) {
```

Comment:
Please add a test for when there are no usage records for an attributionID and it should return a 0

Reply:
Added it [here](https://github.com/gitpod-io/gitpod/pull/13222/files#diff-fa1d370734494a0f6a33ba67a648028093ddd21743ef6df87ee8b9af4410c137R370-R373)